client/src/app/core/repositories/motions/motion-repository.service.ts

```diff
@@ -801,4 +802,25 @@ export class MotionRepositoryService extends BaseAgendaContentObjectRepository<V
             return value;
         }
     }
+
+    /**
+     * Triggers an update for the sort function responsible for the default sorting of data items
+     */
+    public setConfigSortFn(): void {
```

Comment:
Motion default sorting is missing.
The config has the property:
"Sort motions by" which can either be "Identifier" or "Call list"

Should probably look like in the category repo.